templates/projects/progress_reminder.html

```diff
@@ -0,0 +1,20 @@
+{% extends "projects/base.html" %}
+{% set active_page = "projects" %}
+{% set active_link  = "tasks" %}
+{% set section = _('Progress Reminder') %}
```

Comment:
If we decide to rename the page title, we may want to update this as well to "Task Notifications" or similar wording.

The filename could be updated as well from `progress_reminder.html` to `task_notifications.html`. :)